liwords-ui/src/App.tsx

```diff
@@ -159,6 +171,37 @@ const App = React.memo(() => {
     getMods();
   }, [getMods]);
 
+  const getFriends = useCallback(() => {
+    axios
+      .post<FriendsResponse>(
+        toAPIUrl('user_service.SocializeService', 'GetFollows'),
+        {},
+        {}
+      )
+      .then((resp) => {
+        console.log('Fetched friends:', resp);
+        const friends: { [uuid: string]: FriendUser } = {};
+        Object.values(resp.data.users).forEach((f: FriendUser) => {
+          friends[f.uuid] = f;
+        });
+        setFriends(friends);
+      })
+      .catch((e) => {
+        console.log(e);
```

Comment:
it's calling this api when not logged in, and getting 401. (#565 patched the same issue for the blocked list)